File: Modul-3/Soal-5/PRAK305-2410817220012-NOOR-KHALISA.py
totalDetik = int(input("Masukkan jumlah detik: "))

hari = 0
if totalDetik >= 86400:
    hari = totalDetik // 86400
    totalDetik = totalDetik %86400

if totalDetik >= 3600:
    jam = totalDetik // 3600
    totalDetik = totalDetik % 3600
else: jam = 0

if totalDetik >= 60:
    menit = totalDetik // 60
    totalDetik = totalDetik % 60
else: menit = 0

detik = totalDetik
if hari > 0:
    print(f"{hari} hari {jam:02}:{menit:02}:{detik:02}")
else:    
    print(f"{jam:02}:{menit:02}:{detik:02}")

File: Modul-3/Soal-5/PRAK305-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
int main(){
    int total_detik, hari, jam, menit, detik;
    printf("Masukkan jumlah detik: ");
    scanf("%d", &total_detik);

    if (total_detik >= 86400)
    {hari = total_detik / 86400;
    total_detik = total_detik % 86400;}
    else
    {hari = 0;}

    if (total_detik >= 3600)
    {jam = total_detik / 3600;
    total_detik = total_detik % 3600;}
    else
    {jam = 0;}

    if (total_detik >= 60)
    {menit = total_detik / 60;
    total_detik = total_detik % 60;}
    else
    {menit = 0;}

    detik = total_detik;

    if (hari > 0)
    {printf("%d hari %02d:%02d:%02d", hari, jam, menit, detik);}
    else
    {printf("%02d:%02d:%02d", jam, menit, detik);}
    
    return 0;
}